hosts_db.c

```diff
@@ -1250,8 +1250,8 @@ text_metrics_format_host_key(struct str *buf, const struct bucket *b) {
    const char *ip = addr_to_str(&(b->u.host.addr));
 
    str_appendf(buf,
-      "host_bytes_total{interface=\"%s\",ip=\"%s\"",
-      title_interfaces, ip);
+      "host_bytes_total{interface=\"%s\",ip=\"%s\",host=\"%s\"",
+      title_interfaces, ip, b->u.host.dns);
```

Comment:
It's generally not considered good practice to add things like `host` to every metric; this can easily be added by Prometheus on ingestion if desired (see `relabel_configs`). You could also join on another metric to get it. node exporter, for example, has a single metric (`node_uname_info`) which exposes the `domainname` and `nodename` labels.

Reply:
No prob. By the way I'm not a maintainer of this repo, it was just a drive-by comment.

Reply:
No, I appreciate it much. I'm all for efficiency! I guess I had tunnel vision.